docs/security.md

```diff
@@ -258,3 +258,31 @@ If you want to use the command `curl`, you can follow the commands:
+| `gravitino.auxService.iceberg-rest.allowedOrigins`       | A comma separated list of origins that are allowed to access the resources. Default value is *, meaning all origins.                                                                                                                          | `*`                                           | No        | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.allowedTimingOrigins` | A comma separated list of origins that are allowed to time the resource. Default value is the empty string, meaning no origins.                                                                                                               | ``                                            | No        | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.allowedMethods`       | a comma separated list of HTTP methods that are allowed to be used when accessing the resources. Default value is GET,POST,HEAD,DELETE.                                                                                                       | `GET,POST,HEAD,DELETE`                        | No        | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.allowedHeaders`       | A comma separated list of HTTP headers that are allowed to be specified when accessing the resources. Default value is X-Requested-With,Content-Type,Accept,Origin. If the value is a single *, this means that any headers will be accepted. | `X-Requested-With,Content-Type,Accept,Origin` | No        | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.preflightMaxAge`      | The number of seconds that preflight requests can be cached by the client. Default value is 1800 seconds, or 30 minutes.                                                                                                                      | `1800`                                        | No        | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.allowCredentials`     | A boolean indicating if the resource allows requests with credentials. Default value is true.                                                                                                                                                 | `true`                                        | No        | 0.4.0         |
+| `gravitino.auxService.iceberg-rest.exposedHeaders`       | A comma separated list of HTTP headers that are allowed to be exposed on the client. Default value is the empty list.                                                                                                                         | ``                                            | No        | 0.4.0         |
```

Comment:
I noticed that  `Server configuration` and `Iceberg REST service's configuration` are almost the same, can we merge them into one and say 'Iceberg rest are the same, the difference is that the prefix of configuration key is ....'

Reply:
Maybe it's will be difficult for users.